### PR TITLE
AdminDb docs

### PR DESCRIPTION
**Pull request by sviatosluv**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -28,6 +29,7 @@ straightforward as possible.
 - Dumping duplicate files
 - Drive dump failing after 1 minute mark
 - Adaptive uploads and users/rooms list for desctop
+- Error entering room on main drive fix
```
Comment (sergey-lukianov) [inline CHANGELOG.md:32]: Leaked from another PR ?

File: lib/chat/admin_db_structures.livemd [hunk 2]
```diff
@@ -1,5 +1,69 @@
...
+
+```elixir
+{:cargo_settings, CargoSettings{checkpoints: [pub_key1, pub_key2, pub_key3]}}
+```
+
+<!-- livebook:{"break_markdown":true} -->
+
+### Media settings
+
+When new USB drive is plugged into the secondary port,the following functionality will be started.
```
Comment (sergey-lukianov) [inline lib/chat/admin_db_structures.livemd:53]: When empty USB drive ....
Functionalities: 
 - backup - full backup of main drive
 - onliners - backup of data in scope of online (at this moment) users
 - cargo - special cargo scenario

File: lib/chat/admin_db_structures.livemd
Comment (sergey-lukianov) [inline lib/chat/admin_db_structures.livemd:1]: This should be linked in README.md as well

**Review by sergey-lukianov — CHANGES_REQUESTED** [review]

File: lib/chat/admin_db_structures.livemd
Comment (sviatosluv) [reply lib/chat/admin_db_structures.livemd:1]: already done

**Review by sergey-lukianov — APPROVED** [review]

**Pull request merged**